### PR TITLE
Update READMEs for doc building changes

### PR DESCRIPTION
**Pull request by m57lyra**
- Explicitly call out how to skip doc builds for companies who cannot use Docker, and how to run them without Docker separately.
- Remove Known issues, as they are no longer issues
- Remove deprecated sphinx autobuild

### CODE REVIEW

**Comment by m57lyra** [comment]
NOTE: presto-docs builds but my actual presto does not. I don't think it's related to these changes.

Both files view as expected in the github View File interface, and the link I added to ./presto-docs/README.md in ./README works from the github interface.

**Comment by m57lyra** [comment]
Addresses issues in https://github.com/prestosql/presto/issues/4807

**Comment by m57lyra** [comment]
@mosabua - it builds (had to update my jdk). Please review

**Comment by m57lyra** [comment]
added tip for resuming some failed build by leaving out the clean arg

File: README.md [hunk 1]
```diff
@@ -18,50 +18,82 @@
 ## Requirements
 
 * Mac OS X or Linux
-* Java 11, 64-bit
+* Java 11.0.8 or later, 64-bit
```
Comment (mosabua) [inline README.md:21]: thats not true as such since Java 12, 13 , 14.. is not really supported/tested

maybe 

* Java 11, 64-bit (> 11.0.2) 

Not sure to be honest ..  wdyt @electrum ?

File: README.md [hunk 2]
```diff
@@ -18,50 +18,82 @@
...
 ## Building Presto
 
-Presto is a standard Maven project. Simply run the following command from the project root directory:
+Presto is a standard Maven project. Simply run the following command from the
+project root directory:
 
     ./mvnw clean install
 
-On the first build, Maven will download all the dependencies from the internet and cache them in the local repository (`~/.m2/repository`), which can take a considerable amount of time. Subsequent builds will be faster.
+On the first build, Maven will download all the dependencies from the internet
```
Comment (mosabua) [inline README.md:32]: Maven downloads

File: README.md [hunk 3]
```diff
@@ -18,50 +18,82 @@
...
+and cache them in the local repository (`~/.m2/repository`), which can take a
```
Comment (mosabua) [inline README.md:33]: and caches them

File: README.md [hunk 4]
```diff
@@ -18,50 +18,82 @@
...
+considerable amount of time. Subsequent builds will be faster.
```
Comment (mosabua) [inline README.md:34]: are faster.

File: README.md [hunk 5]
```diff
@@ -18,50 +18,82 @@
...
 
-Presto has a comprehensive set of unit tests that can take several minutes to run. You can disable the tests when building:
+Presto has a comprehensive set of unit tests that can also take a considerable
+amount of time to run. You can disable the tests when building:
 
     ./mvnw clean install -DskipTests
 
+If you experience a build fail, in many cases you can resume the build once you
+have corrected the build issue by simply leaving the 'clean' argument out of the
```
Comment (mosabua) [inline README.md:42]: backticks around commands

File: README.md [hunk 6]
```diff
@@ -18,50 +18,82 @@
...
+build command:
+
+    ./mvnw install -DskipTests
+
+If your organization does not allow the use of Docker, you can skip building
```
Comment (mosabua) [inline README.md:47]: Maybe not talk about org .. just talk about .. if you do not have Docker installer...

**Review by mosabua — CHANGES_REQUESTED** [review]
A bunch of changes needed.

File: README.md (hunk 6 above)
Comment (m57lyra) [reply README.md:47]: use

File: README.md (hunk 1 above)
Comment (electrum) [reply README.md:21]: Yes, maybe leave this alone. I think that people have successfully run under JDK 14, but I know for certain that JDK 15 does not work.

File: README.md [hunk 7]
```diff
@@ -18,50 +18,82 @@
 ## Requirements
 
 * Mac OS X or Linux
-* Java 11, 64-bit
+* Java 11.0.8 or later, 64-bit
 * Python 2.6+ (for running with the launcher script)
-* Docker (for building and running tests)
+* Docker (for building and running tests, and for building documentation)
```
Comment (electrum) [inline README.md:23]: The "for building" covers building documentation

File: README.md [hunk 8]
```diff
@@ -18,50 +18,82 @@
...
 
 ## Building Presto
 
-Presto is a standard Maven project. Simply run the following command from the project root directory:
+Presto is a standard Maven project. Simply run the following command from the
```
Comment (electrum) [inline README.md:27]: These line wrapping changes are formatting changes, which should be done as a separate commit that only changes formatting (no content changes). It's hard to review content changes when formatting changes are mixed together.

Comment (m57lyra) [reply README.md:27]: Thank you!

File: README.md [hunk 9]
```diff
@@ -15,53 +15,85 @@
...
+and caches them in the local repository (`~/.m2/repository`), which can take a
+considerable amount of time. Subsequent builds are faster.
 
-Presto has a comprehensive set of unit tests that can take several minutes to run. You can disable the tests when building:
+Presto has a comprehensive set of unit tests that can also take a considerable
+amount of time to run. You can disable the tests when building:
 
     ./mvnw clean install -DskipTests
 
+If you experience a build fail, in many cases you can resume the build once you
```
Comment (electrum) [inline README.md:41]: Let's remove this part, since removing `clean` can be dangerous and doesn't really help anyway

File: README.md (hunk 7 above)
Comment (electrum) [reply README.md:23]: We discussed and agreed to remove the parentheses and just say that Docker is a hard requirement (as various parts of the build use it and we expect more in the future)

File: README.md [hunk 10]
```diff
@@ -15,53 +15,85 @@
...
+build command:
+
+    ./mvnw install -DskipTests
+
+If you do not use Docker, you can skip building the documentation and running
+unit tests by using the following command:
+
+    ./mvnw -pl '!presto-docs' clean install -DskipTests
+
+See the "Default build" section of [presto-docs README](./presto-docs/README.md)
```
Comment (electrum) [inline README.md:52]: We no longer accurate, as we no longer provide instructions for (or support) building documentation without Docker.

Comment (m57lyra) [reply README.md]: removed any reference to building w/o docker

**Comment by m57lyra** [comment]
removed all references to building w/o docker in presto/README. Updated presto-docs/README to no longer use make.

**Comment by electrum** [comment]
Thanks!
